model/src/market/reason.rs

```diff
@@ -0,0 +1,23 @@
+/*
+ * Yagna Market API
+ *
+ * The version of the OpenAPI document: 1.6.1
+ *
+ * Generated by: https://openapi-generator.tech
+ */
+
+use serde::{Deserialize, Serialize};
+
+/// Generic Event reason information structure.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct Reason {
+    #[serde(rename = "message", skip_serializing_if = "Option::is_none")]
+    pub message: Option<String>,
```

Comment:
We can have optional Reason and required Message within or required Reason and optional Message.
I think the latter is more flexible, because we can add other fields in the future (eg. error code) and skip passing Message, by just supplying error code, and this will not break the compatibility. What do you think?

Reply:
I didn't want to stop supporting message. I think about error codes as a way to group terminate reasons. But still someone can include better detailed explanation in message. Error codes will never give us as much flexibility, because we will have to define them ourselfs, otherwise it would be total mess in error codes if anyone could add on their own.